services/grafana/client.go

```diff
@@ -300,7 +303,34 @@ func (c *Client) testDeleteUser(ctx context.Context, userID int, authHeaders htt
 	return c.do(ctx, "DELETE", "/api/admin/users/"+strconv.Itoa(userID), "", authHeaders, nil, nil)
 }
 
-func (c *Client) testCreateAPIKey(ctx context.Context, name string, role role, authHeaders http.Header) (int, string, error) {
+func (c *Client) CreateAdminAPIKey(ctx context.Context, name string) (int, string, error) {
```

Comment:
// CreateAdminAPIKey creates API key to be used by DBaaS clusters

something like that

Reply:
Done